src/vm/excep.cpp

```diff
@@ -7164,6 +7164,13 @@ EXTERN_C void JIT_MemCpy_End();
 
 EXTERN_C void JIT_WriteBarrier_End();
 EXTERN_C void JIT_CheckedWriteBarrier_End();
+#else
+#ifdef FEATURE_PAL
+EXTERN_C void JIT_WriteBarrierGroup();
```

Comment:
I am not sure I understand why you have added these new symbols. There already are JIT_WriteBarrierStart and JIT_WriteBarrierLast  / JIT_PatchedWriteBarrierStart and JIT_PatchedWriteBarrierLast symbols that you can use.